src/include/catalog/namespace.h

```diff
@@ -95,4 +95,6 @@ extern char *namespace_search_path;
 
 extern List *fetch_search_path(bool includeImplicit);
 
+#define HiveProfileName "Hive"
```

Comment:
then I think it should be part of namespace.c - the header file is used by other files, so if it's not useful for them it shouldn't be there.